### DISCUSSION TITLE
Snap7 from windows CMD (Bat file)

### DISCUSSION THREAD

**Discussion by hrravn** [gijzelaerr/python-snap7#253] [Q&A] [answered]
I am using the snap7 and it works perfect in my Thonny, but when i run it as a bat file in windows it has problemsfinding the Snap7 libarary

![image](https://user-images.githubusercontent.com/78913536/107644483-c5c71a00-6c77-11eb-912d-5e8ec6a06a5f.png)

**Comment by swamper123** [comment]
Ahoi,
what is your _path_, were you stored the *.dll files, and is that _path_ in _enviromental variables_?

**Reply by hrravn** [reply, marked as answer]
i was missing the enviromental variables (y) works now
